src/FSharp.Formatting.Common/Templating.fs

```diff
@@ -114,6 +114,10 @@ module ParamKeys =
 
     /// A parameter key known to FSharp.Formatting
     let ``fsdocs-watch-script`` = ParamKey "fsdocs-watch-script"
+    let ``fsdocsMenuHeaderContentKey`` = ParamKey "fsdocs-menu-header-content"
```

Comment:
Please use the exact same format as the existing code.
This is a general guideline for any PR you ever make, look at the existing code and just do the same thing.

```
/// A parameter key known to FSharp.Formatting
let ``fsdocs-menu-header-content`` = ParamKey "fsdocs-menu-header-content"
```